### DISCUSSION TITLE
ILI9225 as display for NodeMCU ESP8266

### DISCUSSION THREAD

**Discussion by MeowCattoNiP** [Bodmer/TFT_eSPI#2479] [Q&A - General]
So I am creating a WiFi scanner that displays the various information of a network like SSID and stuff. How can I use the ILI9225 as a display to show the details of scanned networks? Yes I realise that using this screen is overkill but what the hell haha. Currently my connection is:

Screen -> ESP8266
VCC -> 3V3
GND -> GND
CLK -> D5
SDA -> D7
RS -> D2
RST -> D4
CS -> D8

**Comment by Bodmer** [comment]
I assume the display is connected and working?

**Reply by MeowCattoNiP** [reply]
yea sure its connected, but i cant tell if its working or not? I dont have an arduino to help me test it out. The screen lit up tho so maybe its working??
